### DISCUSSION TITLE
Take time to explain what setup needs for "enter your IPv4 address with CIDR mast"

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#5132] [Ideas] [answered]
Hi, I am using securityonion-2.3.61-STENODOCKER.iso. In your installation guides perhaps you can explain what "enter your IPv4 address with CIDR mast" means, where to find this IPv4 address, what it does, what it affects and issues for entering the wrong one. 
  I dont expect you to explain all of this in the prompt itself when you get to this point of the setup, but when you give installation and setup steps on your website, you could slip this in to the security onion documentation where it explains the steps to setup SO?
  Also,  at some point it also asks "enter a single IP address or an IP range, in CIDR notation to allow". Not sure what IP address should go into this last prompt?
   Maybe you could provide the same type of documentation for this on your setup instructions on the SO website?  
  It would especially help those new to SO.
  Just an idea, Thanks

**Comment by dougburks** [comment, marked as answer]
We are improving the verbiage for these two prompts:
https://github.com/Security-Onion-Solutions/securityonion/issues/5138

However, please keep in mind that it's still your responsibility to understand your network, what IP address you want to assign to the Security Onion installation, and what IP address(es) you want to allow to connect to the Security Onion installation.
